### ISSUE TITLE
Extensions documentation is ignored

### ISSUE THREAD

**Issue by MattCheetham** [realm/jazzy#454]
I have an extension of NSDate in my project. It looks a little like this

```
/**
Useful extensions of NSDate that do things
*/
public extension NSDate {

}
```

Jazzy puts the NSDate extension into the documentation when generated but lists it as "Undocumented".

Not sure if I'm missing something obvious or of it's a bug.

**Comment by juliangrosshauser** [comment]
I'm having the same problem in an Obj-C project.

This category ends up being "Undocumented":

```objc
/// See PSPDFPresentationActions.h for compatible keys for `options`.
@interface PSPDFViewController (Presentation)
```

Whereas this category ends up being documented:

```objc
/// This defines shortcuts that will edit the `fontAttributes` dictionary.
/// Valid for PSPDFFreeTextAnnotation, PSPDFChoiceFormElement and PSPDF
@interface PSPDFAnnotation (Fonts)
```

**Comment by jverkoey** [comment]
I'm also seeing anonymous categories incorrectly called out for being undocumented.

```
/** Some category */
@interface SomeClass ()

/** Some docs */
@property(nonatomic, copy) NSArray *anArray;

@end
```

Both `SomeClass` and `SomeClass.anArray` are marked as undocumented.

**Comment by JoshHrach** [comment]
I'm seeing this as well. I made an extension of UIDevice, provided documentation on it, but it also is ignored when skipping undocumented items.

**Comment by istx25** [comment]
Any update on this?

**Comment by jpsim** [comment]
It'd be useful to check to see if this is still an issue.

**Comment by JoshHrach** [comment]
I generated documentation last Wednesday. We have extensions for Date, UILabel, UIDevice, and UIViewController. All have the same level of documentation. Only the Date extension is showing up in our documentation.

**Comment by jpsim** [comment]
@JoshHrach is this with jazzy 0.7.3? What steps exactly could I follow to trigger this issue?

**Comment by jpsim** [comment]
Fixed in #819
